src/main/java/com/suse/saltstack/netapi/datatypes/StartTime.java

```diff
@@ -0,0 +1,65 @@
+package com.suse.saltstack.netapi.datatypes;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+import com.suse.saltstack.netapi.exception.ParsingException;
+
+/**
+ * StartDate is a convenience wrapper allowing for parsing of StartDate in the
+ * timezone appropriate to a given master.
+ */
+
+public class StartTime {
+    private final String dateString;
+
+    // StartTime example from API: "2015, Mar 04 19:28:29.724698"
+    private transient final SimpleDateFormat startJobDateFormat =
+            new SimpleDateFormat("yyyy, MMM dd HH:mm:ss.SSS", Locale.US);
+
+    public StartTime(String dateString) throws ParsingException {
+        try {
+            if (dateString != null)
+                startJobDateFormat.parse(dateString);
+        } catch (ParseException e) {
+            throw new ParsingException(e);
+        }
+        this.dateString = dateString;
+    }
+
+    /**
+     * Returns @{link Date} representation of StartTime as parsed at a given timezone.
+     * Master does not return a timezone associated with StartTime timestamp string,
+     * therefore an explicit timezone needs to be provided for correct parsing.
+     *
+     * @param tz TimeZone associated with master.
+     * @return @{link Date} representation of StartTime at provided timezone
```

Comment:
The `@` needs to go inside the `{}` here (and above), otherwise javadoc will fail thinking that this is the next tag, see [travis] (https://travis-ci.org/SUSE/saltstack-netapi-client-java/builds/79381432#L1574).